### DISCUSSION TITLE
Fine Tuning VITS model gives robotic metallic sound

### DISCUSSION THREAD

**Discussion by Muhammad057** [coqui-ai/TTS#2888] [Qs for Live Q&A]
Hello everyone, I have 145 training samples of around 3 seconds to 14 seconds each. I fine-tined the VITS model. The voice similarity is okay but it gives me a metallic noisy voice. Any ideas on what can I do to avoid the metallic output?
1. Should I increase the training dataset size?
2. Should I fine-tune yourTTS model and see the results?

Thank you.

**Comment by Harrolee** [comment]
Check out this [thread](https://github.com/coqui-ai/TTS/discussions/2507#discussioncomment-6913500)
I had the same result early on in my training process. It gets better with more iterations.
